migen/genlib/fifo.py

```diff
@@ -182,8 +194,9 @@ class AsyncFIFO(Module, _FIFOInterface):
     """
     __doc__ = __doc__.format(interface=_FIFOInterface.__doc__)
 
-    def __init__(self, width, depth):
-        _FIFOInterface.__init__(self, width, depth)
+    def __init__(self, width, depth, fwft=False):
```

Comment:
``fwft`` does not make sense for an async FIFO and therefore this parameter should not be there.